app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'views/form',
+  'stache!templates/partial/coppa-age-input',
+  'lib/strings',
+  'lib/auth-errors',
+  'lib/promise',
+  'views/mixins/floating-placeholder-mixin'
+], function (Cocktail, FormView, Template, Strings, AuthErrors, p, FloatingPlaceholderMixin) {
+  'use strict';
+
+  var CUTOFF_AGE = 13;
+
+  var View = FormView.extend({
+
+    template: Template,
+    className: 'coppa-age-input',
+
+    initialize: function (options) {
+      options = options || {};
+    },
+
+    events: {
+      'input': 'onInput'
+    },
+
+    /**
+     * Called by the parent view to determine if the user is old enough.
+     * @method isUserOldEnough
+     * @param {Object} [userAge] - hash with `year`, `month`, and `date`. Used
+     * for testing.
+     * @return {Boolean} true if user is old enough, false otw.
+     */
+    isUserOldEnough: function (userAge) {
+      return this._getAge(userAge) >= CUTOFF_AGE;
+    },
+
+    onInput: function (e) {
+      var val = this.$('#age').val();
+      this.$('#age').val(val.substr(0, 3));
+    },
+
+    context: function () {
+      return {};
+    },
+
+    render: function () {
+      var self = this;
+
+      return p()
+        .then(function () {
+          self.$el.html(self.template(self.getContext()));
+
+          self.initializePlaceholderFields();
+
+        });
+    },
+
+    /**
+     * Called by the parent view to show any validation errors.
+     *
+     * @method showValidationErrorsEnd
+     */
+    showValidationErrorsEnd: function () {
+      console.log('HELLO ANYONE HERE??');
```

Comment:
@shane-tomlinson any idea why is not getting called?

![omgcoppa](https://cloud.githubusercontent.com/assets/128755/9322046/36fd13fa-453c-11e5-808a-e1afec060e52.gif)

Most of the stuff works, it's just the "disabled" submit behaviour is not showing an error. What am I missing?